src/api/app/views/webui/project/_incident_table_entries.html.erb

```diff
@@ -42,7 +42,7 @@
                 <% requests_out = BsRequest.list(roles: %w(source), states: %w(new review declined),
                                                           types: %w(maintenance_release), project: incident.name) %>
                 <% if requests_out.present? %>
-                    <% requests_out.order('bs_requests.number').each do |rq_out| %>
+                    <% requests_out.sort {|x,y| x.number <=> y.number}.each do |rq_out| %>
```

Comment:
Right, but this PR is not about rewriting the whole BsRequest collection stuff;-)